<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiniPass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuOperaciones">
    <property name="title">
     <string>Configuracion</string>
    </property>
    <addaction name="actionControladores"/>
    <addaction name="actionPuertas"/>
    <addaction name="actionNiveles_de_acceso"/>
    <addaction name="actionUsuarios"/>
    <addaction name="actionEntradas"/>
    <addaction name="actionSalidas"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionSobre"/>
   </widget>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Operaciones</string>
    </property>
    <addaction name="actionRegistros"/>
    <addaction name="actionConectar_sistema"/>
    <addaction name="actionSystemUpdate"/>
    <addaction name="actionLimpiar_eventos"/>
   </widget>
   <addaction name="menuOperaciones"/>
   <addaction name="menuOpciones"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionControladores"/>
   <addaction name="actionPuertas"/>
   <addaction name="actionUsuarios"/>
   <addaction name="separator"/>
   <addaction name="actionRegistros"/>
   <addaction name="actionConectar_sistema"/>
   <addaction name="actionSystemUpdate"/>
  </widget>
  <action name="actionControladores">
   <property name="icon">
    <iconset resource="MiniPass.qrc">
     <normaloff>:/new/prefix1/resourses/tarjeta_expansion.png</normaloff>:/new/prefix1/resourses/tarjeta_expansion.png</iconset>
   </property>
   <property name="text">
    <string>Controladores...</string>
   </property>
  </action>
  <action name="actionNiveles_de_acceso">
   <property name="icon">
    <iconset resource="MiniPass.qrc">
     <normaloff>:/new/prefix1/resourses/security_agent.png</normaloff>:/new/prefix1/resourses/security_agent.png</iconset>
   </property>
   <property name="text">
    <string>Niveles de acceso...</string>
   </property>
  </action>
  <action name="actionUsuarios">
   <property name="icon">
    <iconset resource="MiniPass.qrc">
     <normaloff>:/new/prefix1/resourses/id_card.png</normaloff>:/new/prefix1/resourses/id_card.png</iconset>
   </property>
   <property name="text">
    <string>Usuarios...</string>
   </property>
  </action>
  <action name="actionRegistros">
   <property name="icon">
    <iconset resource="MiniPass.qrc">
     <normaloff>:/new/prefix1/resourses/disk.png</normaloff>:/new/prefix1/resourses/disk.png</iconset>
   </property>
   <property name="text">
    <string>Registros...</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="text">
    <string>Sobre...</string>
   </property>
  </action>
  <action name="actionSystemUpdate">
   <property name="icon">
    <iconset resource="MiniPass.qrc">
     <normaloff>:/new/prefix1/resourses/DB Update.png</normaloff>:/new/prefix1/resourses/DB Update.png</iconset>
   </property>
   <property name="text">
    <string>Actualizar sistema</string>
   </property>
  </action>
  <action name="actionPuertas">
   <property name="icon">
    <iconset resource="MiniPass.qrc">
     <normaloff>:/new/prefix1/resourses/puerta-icono-6807-32.png</normaloff>:/new/prefix1/resourses/puerta-icono-6807-32.png</iconset>
   </property>
   <property name="text">
    <string>Puertas...</string>
   </property>
  </action>
  <action name="actionLimpiar_eventos">
   <property name="text">
    <string>Limpiar eventos</string>
   </property>
  </action>
  <action name="actionEntradas">
   <property name="text">
    <string>Entradas...</string>
   </property>
  </action>
  <action name="actionSalidas">
   <property name="text">
    <string>Salidas..</string>
   </property>
  </action>
  <action name="actionConectar_sistema">
   <property name="icon">
    <iconset resource="MiniPass.qrc">
     <normaloff>:/new/prefix1/resourses/20131203025132621_easyicon_net_32.png</normaloff>:/new/prefix1/resourses/20131203025132621_easyicon_net_32.png</iconset>
   </property>
   <property name="text">
    <string>Conectar sistema</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MiniPass.qrc"/>
 </resources>
 <connections/>
</ui>
